server/main.py

```diff
@@ -56,115 +55,66 @@ def __init__(self) -> None:
 
         It stores the active connections and is able to broadcast data.
         """
-        self._active_connections: set[Client] = set()
+        self._rooms: dict[str, set[Client]] = {}
```

Comment:
Perhaps instead of a dict as the data structure, use a pydantic model? They're used a lot for FastAPI

Reply:
This was solved with #16 .